azure-mgmt-trafficmanager/src/main/java/com/microsoft/azure/management/trafficmanager/TrafficManagerEndpoint.java

```diff
@@ -0,0 +1,500 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.trafficmanager;
+
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.resources.fluentcore.arm.Region;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.ExternalChildResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Attachable;
+import com.microsoft.azure.management.resources.fluentcore.model.Settable;
+import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
+import com.microsoft.azure.management.trafficmanager.implementation.EndpointInner;
+
+/**
+ * An immutable client-side representation of an Azure traffic manager profile endpoint.
+ */
+@Fluent
+public interface TrafficManagerEndpoint extends
+        ExternalChildResource<TrafficManagerEndpoint, TrafficManagerProfile>,
+        Wrapper<EndpointInner> {
+    /**
+     * @return the endpoint type
+     */
+    EndpointType endpointType();
+
+    /**
+     * @return the monitor status of the endpoint
+     */
+    EndpointMonitorStatus monitorStatus();
+
+    /**
+     * @return true if the endpoint is disabled, false otherwise
+     */
+    boolean isDisabled();
```

Comment:
regardless of what the Inner APis say, in general, API and UI design guidance MS usually follows is to word boolean option naming in the positive, not negative, i.e.: isEnabled(), not isDisabled()